components/src/core/UserMenu/UserMenu.tsx

```diff
@@ -188,7 +199,7 @@ export const UserMenu: React.FC<UserMenuProps> = (props) => {
     }, [menu, anchorEl, closeMenu, MenuProps, printMenu]);
 
     return (
-        <div className={clsx(defaultClasses.root, classes.root)}>
+        <div className={clsx(defaultClasses.root, classes.root)} {...otherDivProps}>
```

Comment:
This root `div` element doesn't really do anything. I'm more inclined to replace it with `<></>` and have the `Avatar` be the root element and spread props to that.

Reply:
or maybe in this case we take `AvatarProps` and `MenuProps`

Reply:
We have MenuProps already. The user passes in their own `<Avatar />`, which means they have full control over the AvatarProps.

Reply:
good point. Alright, it's fine with me then. A little redundant, but that's okay